.travis.yml

```diff
@@ -49,7 +49,7 @@ matrix:
     - cargo build
     - cargo test
     - yarn test
-  - rust: nightly-2017-03-04
+  - rust: nightly-2017-06-29
```

Comment:
This is related to updating clippy. Old clippy (0.0.118) depended on rustc-serialize and new one (0.0.141) won't build with nightly-2017-03-04.